### ISSUE TITLE
How to Solve Xcode linker command failed with exit code 1 error?

### ISSUE THREAD

**Issue by Tawfikment** [unity-plugins/Unity-Admob#138]
I build my game from Unity to Xcode, the game has GameAnalytics and GoogleMobileAds SDK, I opened the .xcworkspace file with Xcode (Version 9.2), but when I try building it (for Simulator and also for real device) I always got this error:

> Ld /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Products/ReleaseForRunning-iphonesimulator/YourApp.app/YourApp normal x86_64 cd "/Users/Taik/Desktop/Unity Games/YourGame/Xcode Simulator" export IPHONEOS_DEPLOYMENT_TARGET=8.0 export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator11.2.sdk -L/Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Products/ReleaseForRunning-iphonesimulator -L/Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Products/ReleaseForRunning-iphonesimulator/GoogleUtilities -L/Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Products/ReleaseForRunning-iphonesimulator/nanopb -L/Users/Taik/Desktop/Unity\ Games/Your\ Game\ 3D/Xcode\ Simulator -L/Users/Taik/Desktop/Unity\ Games/Your\ Game\ 3D/Xcode\ Simulator/Libraries -L/Users/Taik/Desktop/Unity\ Games/Your\ Game\ 3D/Xcode\ Simulator/Libraries/GameAnalytics/Plugins/iOS -F/Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Products/ReleaseForRunning-iphonesimulator -F/Users/Taik/Desktop/Unity\ Games/Your\ Game\ 3D/Xcode\ Simulator/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current -F/Users/Taik/Desktop/Unity\ Games/Your\ Game\ 3D/Xcode\ Simulator/Pods/GoogleAppMeasurement/Frameworks -filelist /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Intermediates.noindex/Unity-iPhone.build/ReleaseForRunning-iphonesimulator/Unity-iPhone.build/Objects-normal/x86_64/YourApp.LinkFileList -Xlinker -map -Xlinker /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Intermediates.noindex/Unity-iPhone.build/ReleaseForRunning-iphonesimulator/Unity-iPhone.build/YourApp-LinkMap-normal-x86_64.txt -mios-simulator-version-min=8.0 -dead_strip -Xlinker -object_path_lto -Xlinker /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Intermediates.noindex/Unity-iPhone.build/ReleaseForRunning-iphonesimulator/Unity-iPhone.build/Objects-normal/x86_64/YourApp_lto.o -Xlinker -objc_abi_version -Xlinker 2 -stdlib=libc++ -fobjc-arc -fobjc-link-runtime -ObjC -lGoogleUtilities -lc++ -lnanopb -lsqlite3 -lz -framework AVFoundation -framework AudioToolbox -framework CFNetwork -framework CoreGraphics -framework CoreMedia -framework CoreTelephony -framework CoreVideo -framework GoogleAppMeasurement -framework GoogleMobileAds -framework MediaPlayer -framework MessageUI -framework MobileCoreServices -framework QuartzCore -framework Security -framework StoreKit -framework SystemConfiguration -weak_framework AdSupport -weak_framework JavaScriptCore -weak_framework SafariServices -weak_framework WebKit -weak_framework CoreMotion -weak-lSystem -Wl,-undefined,dynamic_lookup -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Intermediates.noindex/Unity-iPhone.build/ReleaseForRunning-iphonesimulator/Unity-iPhone.build/YourApp.app.xcent -framework Security -framework MediaToolbox -liPhone-lib -framework CoreText -framework AudioToolbox -weak_framework AVFoundation -framework CFNetwork -framework CoreGraphics -framework CoreMedia -weak_framework CoreMotion -framework CoreVideo -framework Foundation -framework MediaPlayer -framework OpenAL -framework OpenGLES -framework QuartzCore -framework SystemConfiguration -framework UIKit -liconv.2 -lil2cpp -lGameAnalytics -weak_framework Metal -lsqlite3 -lz -framework AdSupport -lPods-Unity-iPhone -Xlinker -dependency_info -Xlinker /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Intermediates.noindex/Unity-iPhone.build/ReleaseForRunning-iphonesimulator/Unity-iPhone.build/Objects-normal/x86_64/YourApp_dependency_info.dat -o /Users/Taik/Library/Developer/Xcode/DerivedData/Unity-iPhone-dktpugebmfsrilfvhywuibshbhep/Build/Products/ReleaseForRunning-iphonesimulator/YourApp.app/YourApp
> 
> ld: warning: -undefined dynamic_lookup is deprecated on iOS ld: framework not found CoreServices for architecture x86_64 clang: error: linker command failed with exit code 1 (use -v to see invocation)

I'm sure that the admob SDK is the reason of this issue, because I tried deleting it and the project works fine, Can you please tell me what's wrong? and how to solve it.

**Comment by unity-plugins** [comment]
CoreServices not support simulator.

**Comment by Tawfikment** [comment]
Yeah but I've already tried with real device, and the same error happen.
